parts/linux/cloud-init/artifacts/cse_config.sh

```diff
@@ -18,49 +18,9 @@ configureSecrets(){
     chmod 0600 "${CA_PRIVATE_KEY_PATH}"
     chown root:root "${CA_PRIVATE_KEY_PATH}"
 
-    ETCD_SERVER_PRIVATE_KEY_PATH="/etc/kubernetes/certs/etcdserver.key"
```

Comment:
you can also remove /etc/kubernetes/certs/apiserver.key and /etc/kubernetes/certs/ca.key

Reply:
in fact we can remove the whole configureSecrets and the call into it